pool/pool_manager_test.gno

```diff
@@ -132,6 +132,7 @@ func TestApiGetPools(t *testing.T) {
 	uassert.Equal(t, response.Size(), 2)
 }
 
-func gnsBalance(addr std.Address) uint64 {
+func gnsBalance(t *testing.T, addr std.Address) uint64 {
```

Comment:
what is meaning of using T here, it's really unnecessary I think.

Reply:
just for calling the `t.Helper`, to indicate that it is a helper function for testing.